plugins/gcc/options-base.js

```diff
@@ -9,15 +9,14 @@ Available Error Groups: accessControls, checkPrototypalTypes,
     checkRegExp, checkTypes, checkVars, conformanceViolations, const,
     constantProperty, deprecated, deprecatedAnnotations, duplicateMessage,
     es5Strict, externsValidation, functionParams, globalThis, invalidCasts,
-    misplacedTypeAnnotation, missingGetCssName, missingOverride,
-    missingPolyfill, missingProperties, missingProvide, missingRequire,
-    missingReturn, missingSourcesWarnings, moduleLoad, msgDescriptions,
+    misplacedTypeAnnotation, missingOverride, missingPolyfill,
+    missingProperties, missingProvide, missingRequire, missingReturn,
+    missingSourcesWarnings, moduleLoad, moduleImports, msgDescriptions,
```

Comment:
Looks like you added `moduleImports` here, does that have any effects on our modularization work from a compiler error perspective?

Reply:
This is in a comment block listing the available error groups from the compiler's `--help` output, which we stuck at the top as a reference. I didn't change the actual groups the resolver adds to the GCC options other than removing `useOfGoogBase` because it doesn't exist in the latest compiler.

We should look into the new options at some point to see if we want to add them, but the goal of this PR was to update options so they work with the new compiler.